### DISCUSSION TITLE
Label definitions

### DISCUSSION THREAD

**Discussion by togfoxy** [togfoxy/MarsLander#201] [Ideas]
To me:

- bug: means runtime error or something is not working as designed. A feature or improvement that is not yet implemented is not a bug.
- enhancement: means something that will improve the code is player experience
- question: means someone is looking for an idea or validate their own idea
- documentation: means the wiki, user guide, release notes or other non-code item needs attention
- graphics: means the provisioning or adjusting of assets (not love.graphics.line  etc)
- audio: means the provisioning or adjusting of audio files

@Philbywhizz @MadByteDE 

Will put into wiki when ready

**Comment by Philbywhizz** [comment]
I think those labels are reasonable
